examples/pygsflow1.1.0_Builder_tutorial_09_land_cover_and_soils.ipynb

```diff
@@ -616,7 +616,7 @@
     "\n",
     "# gravity reservoir routing coefficients\n",
     "ssr2gw_rate = bu.ssr2gw_rate(ksat, sand, soil_moist_max.values)\n",
-    "ssr2gw_sq = bu.ssr2gw_exp(ml.modelgrid.nnodes)\n",
+    "ssr2gw_sq = bu.ssr2gw_exp(int(ml.modelgrid.nnodes/ml.modelgrid.nlay))\n",
```

Comment:
instead of `int(ml.modelgrid.nnodes/ml.modelgrid.nlay)`, this could be more concisely be written as:
`int(ml.modelgrid.ncpl)`